test/JDBC/upgrade/14_5/SYSTEM_USER_dep-vu-prepare.mix

```diff
@@ -0,0 +1,21 @@
+-- tsql
```

Comment:
This doesn't seem to be right, why these prepare verify files are not inside `preparation` directory. There should a preparation directory for 14_5 and all the prepare files should go there.